standalone/install.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env sh
+
+# Install porkcron as a standalone executable script
+cd "$(dirname "$0")"
+cp ../porkcron.sh /usr/local/bin/porkcron
+chmod +x /usr/local/bin/porkcron
+
+mkdir -p /etc/porkcron
+cp ../porkcron.py /etc/porkcron/porkcron.py
+chmod +x /etc/porkcron/porkcron.py
+
+# Add a cron entry for it to run on reboot and weekly
+echo '0 0 * * 1 root /usr/local/bin/porkcron' > /etc/cron.d/porkcron.weekly
+echo '@reboot root /usr/local/bin/porkcron' >> /etc/cron.d/porkcron.reboot
```

Comment:
Can we move the crontab file from /docker to the root and reuse for both methods instead? 🤔

Reply:
I specifically made it separate output files here due to not all flavors of cron supporting the `@periodic` directives. Old dcron/crond installs wouldn't be able to parse it, cronie [needs to be built to do so](https://bugs.gentoo.org/551352), and I'm not sure what e.g. systemd-cron and similar wrappers/converters do to them (the current AUR version is at least a few versions behind).

That's also why I didn't put them in `/etc/cron.{weekly,reboot}/`. Some cron implementations don't read those by default, `/etc/cron.d/` is the only place they all agree on

Reply:
Taking a quick peek at the used docker container and I was apparently wrong. Alpine defaults to Busybox crond and [they have yet another handling of cron paths](https://github.com/mirror/busybox/blob/371fe9f71d445d18be28c82a2a6d82115c8af19d/miscutils/crond.c#L54). And the currently specified directory for the crontab doesn't match anything in those sources.  
I could `cat` them together in a single file during the run step, but I'm not sure if that works and I don't have a machine to test that on